scripts/lib/CIME/get_timing.py

```diff
@@ -65,13 +65,13 @@ def gettime2(self, heading_padded):
 
         heading = '"' + heading_padded.strip() + '"'
         for line in self.finlines:
-            m = re.match(r'\s*{}\s*(\d+)\s*\d+\s*(\S+)'.format(heading), line)
+            m = re.match(r'\s*{}\s+\S\s+(\d+)\s*\d+\s*(\S+)'.format(heading), line)
```

Comment:
@jedwards4b , please take a look at this.

Reply:
Was there a change in the file format?  This seems to match a different string than before - any documentation on this change?

Reply:
@worleyph

Reply:
https://github.com/E3SM-Project/E3SM/pull/2537

Reply:
Is the pull request pointer sufficient to explain this? It makes the per process output file and global statistics file format consistent by adding an "on" column to the global statistics file.

Reply:
@worleyph yes once I found it.